clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -161,44 +164,46 @@ public NetworkClient(Selectable selector,
              discoverBrokerVersions,
              apiVersions,
              null,
-             logContext);
+             logContext,
+             null);
     }
 
     public NetworkClient(Selectable selector,
-            Metadata metadata,
-            String clientId,
-            int maxInFlightRequestsPerConnection,
-            long reconnectBackoffMs,
-            long reconnectBackoffMax,
-            int socketSendBuffer,
-            int socketReceiveBuffer,
-            int defaultRequestTimeoutMs,
-            ClientDnsLookup clientDnsLookup,
-            Time time,
-            boolean discoverBrokerVersions,
-            ApiVersions apiVersions,
-            Sensor throttleTimeSensor,
-            LogContext logContext) {
+        Metadata metadata,
```

Comment:
can you acoid the whitespace changes? would make rebasing easier

Reply:
sure, let me fix